### ISSUE TITLE
Feature of Multiple correct option in the quiz template.

### ISSUE THREAD

**Issue by octacode** [BuildmLearn/BuildmLearn-Toolkit-Android#199]
@opticod @croozeus 
We can add a feature of multiple correct answers in the Quiz Template. It will increase the vastness of the questions the teachers can put in the Quiz Template.
If you find this issue relevant enough. Should I proceed to add this enhancement. Here's a pic depicting how the template will look after the feature has been added.
Thanks.
 
![screenshot_20161031-013237](https://cloud.githubusercontent.com/assets/20819968/19839553/13d253ee-9f0a-11e6-8c0e-d1273d9e4bfd.png)

**Comment by sm86** [comment]
@croozeus @opticod Can I work on this issue stated here.

**Comment by octacode** [comment]
@sm86 I am already done with the code. If it is approved I will send the PR. thanks

**Comment by opticod** [comment]
@croozeus, your views on this issue for our next version?

**Comment by shikharbhuddi** [comment]
@octacode Are you planning to use CheckBox instead of RadioGroup?

**Comment by octacode** [comment]
@shikharbhuddi I am using RadioGroup only but the question having multiple correct answers will be marked with an asterisk.

**Comment by opticod** [comment]
@octacode , you can proceed. But before creating the PR, please discuss your complete implementation.

**Comment by opticod** [comment]
@octacode, check box will be good for multiple correct questions and radio buttons for single correct questions. Marking asterisk will also help, hence add it to editor, simulator and created apks. You can proceed. Please also create PR to templates repo for the same.

**Comment by octacode** [comment]
@opticod, yeah this implementation sounds better. I will make the changes you have suggested and will send a PR.
